tests/UI/pages/BO/shopParameters/search/index.js

```diff
@@ -48,7 +49,16 @@ class Search extends BOBasePage {
     // Columns selectors
     this.tableColumnAliases = row => `${this.tableBodyColumn(row)}:nth-child(2)`;
     this.tableColumnSearch = row => `${this.tableBodyColumn(row)}:nth-child(3)`;
-    this.tableColumnStatus = row => `${this.tableBodyColumn(row)}:nth-child(4)`;
+    this.tableColumnStatus = row => `${this.tableBodyColumn(row)}:nth-child(4) a`;
+
+    // Bulk actions selectors
+    this.bulkActionBlock = 'div.bulk-actions';
+    this.bulkActionMenuButton = '#bulk_action_menu_alias';
+    this.bulkActionDropdownMenu = `${this.bulkActionBlock} ul.dropdown-menu`;
+    this.selectAllLink = `${this.bulkActionDropdownMenu} li:nth-child(1)`;
```

Comment:
I know you probably wrote one million of tests using nth-child but couldn't we make this more robust with new classes? It means that if we add another item in bulkaction (for any reasons), in the middle of 1-7 range or even at the beginning, we'll break a lot of tests

This is a jus a suggestion

Reply:
It's more difficult to found (for me) the file to change to add class for legacy pages. But We will see what we can do.